test/evergreen.yml

```diff
@@ -216,22 +216,23 @@ functions:
           set -o errexit
           set -o verbose
 
-          if [[ "${branch_name}" != "develop" ]]; then
-            echo "We only run the documentation update task on the WiredTiger (develop) Evergreen project."
-            exit 0
+          # Check whether specific branches have been provided to the function
```

Comment:
I feel it won't be so obvious to readers (including our future selves) that `${branch_name}` is an Evergreen built-in expansion that will return the branch configured for the specific Evergreen project in which the test is executed. And instead, `${branches}` is a customized expansion on the build variant level. Can we add some description in the comment?

Reply:
I think it's enough to say that `${branch_name}` is a built-in Evergreen expansion and `${branches}` is a task variable defined in `doc_update`. I'd rename `${branches}` to `${doc_update_branches}`, this makes it more obvious that they are specific to the task and also lets you define local variables in a simpler way (I found the variable name `wt_branches` a bit odd).

Reply:
I will update the variable names and add more details in the comment.

Reply:
See e524b509f375e12baafa1973c983c1178d5207e2